[16.0][ADD] crm referred by categorized (#591)
OCA/crm: crm_referred/readme/DESCRIPTION.md

> 
> This module extends the functionality of crm and to allow you to categorise lead or opportunity according to referred by. Also copy this categorized to partner when partner is created from a lead/opportunity.

Reviewer: ```suggestion
This module extends the functionality of crm and to allow you to categorize lead or opportunity according to referrer. Also copies this category to partners when created from a lead/opportunity.
```
